modules/sandboxed-containers-deleting-pods-cli.adoc

```diff
@@ -0,0 +1,29 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+:_content-type: PROCEDURE
+[id="sandboxed-containers-deleting-pods-cli_{context}"]
+= Deleting {sandboxed-containers-first} pods using the CLI
+
+To uninstall {sandboxed-containers-first}, you must first delete all running pods that use `kata` as the `runtimeClassName`.
```

Comment:
I would say "that use `kata` as the `RuntimeClass`, since the `runtimeClassName` is a definition field and while it _would_ say `kata` there, we aren't using kata as the _name_ of the class, but as the class itself.